### PR TITLE
OSDOCS-2171: RN for tcpdump from must-gather

### PR DESCRIPTION
**Pull request by mikemckiernan**
Please review [Enhancements to must-gather to simplify network tracing](https://deploy-preview-40360--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-networking)

### CODE REVIEW

**Comment by mikemckiernan** [comment]
/assign @alexanderConstantinescu 

This is the release note PR that corresponds to #39615 .  Please let me know any comments or corrections.

**Comment by mikemckiernan** [comment]
/assign @memodi 

Thanks for the review of #39615.  This is the release note PR.  Please let me know any comments or corrections.

**Comment by jboxman-rh** [comment]
cc @jboxman-rh

**Review by alexanderConstantinescu — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -110,6 +110,16 @@ The metric names that start with `ovn` are unique to the OVN-Kubernetes CNI netw
 
 The `ovnkube_master_resource_update_total` metric is removed for the 4.10 release.
 
+[id="ocp-4-10-networking-must-gather-tcpdump"]
+==== Enhancement to must-gather to simplify network tracing
+
+The `oc adm must-gather` command is enhanced to simplify collecting network packet captures.
+You can collect packet captures on multiple nodes at the same time.
+A new `--node-selector` argument provides a way to identify the nodes for the packet captures.
```
Comment (ctauchen) [inline release_notes/ocp-4-10-release-notes.adoc:118]: "for the packet captures"
I found this a bit unclear. Perhaps something like thi? "to identify which nodes you're collecting packet captures for."

Comment (ctauchen) [reply release_notes/ocp-4-10-release-notes.adoc:118]: This note might be more helpful if you mention explicitly what's changed. Something like: It used to be that you could only do this on one node at a time, and now you can do bazillions!

**Review by ctauchen — COMMENTED** [review]
Two suggestions, otherwise lgtm.

**Pull request merged**